src/lib/error/permission-error.ts

```diff
@@ -15,7 +15,7 @@ class PermissionError extends UnleashError {
         const permissionsMessage =
             permissions.length === 1
                 ? `the "${permissions[0]}" permission`
-                : `all of the following permissions: ${permissions
+                : `one of the following permissions: ${permissions
```

Comment:
Boyscouting: I think "one" instead of "all" is more honest and reflective of the real behavior.